### PR TITLE
Deprecate install-prereqs and install-openenclave-stack scripts

### PR DESCRIPTION
**Pull request by oprinmarius**
Deprecate bash scripts
Added any_errors_fatal: true to all ansible playbooks
Added ansible wrapper to be able to include any task list
Updated documentation

### CODE REVIEW

File: scripts/install-prereqs [hunk 1]
```diff
@@ -3,93 +3,22 @@
...
-        apt-get update
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "!!! WARNING THIS SCRIPT WILL BE DEPRECATED IN FAVOUR OF ANSIBLE-PLAYBOOK !!!"
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+command -v "ansible-playbook" > /dev/null
+if [[ $? != 0 ]] 
+then 
+    echo "Ansible is not installed on your system."
+    read -p "Do you want to install it and run the install-prereqs playbook ? " -n 1 -r
```
Comment (johnkord) [inline scripts/install-prereqs]: Personally, I don't really like scripts that prompt the user for things, especially if those scripts might be used in automation!  Maybe we should change the behavior to something like: "if ansible isn't installed, then fail with a message saying that ansible is a dependency, and then tell the user how to install it".

Comment (oprinmarius) [reply scripts/install-prereqs]: @johnkord I applied the suggested changes

File: scripts/install-open-enclave-stack [hunk 2]
```diff
@@ -3,36 +3,16 @@
...
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "!!! WARNING THIS SCRIPT WILL BE DEPRECATED IN FAVOUR OF ANSIBLE-PLAYBOOK !!!"
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+command -v "ansible-playbook" > /dev/null
+if [[ $? != 0 ]]
+then
+    echo "Ansible is required to run this script but it is not installed on your system."
+    echo "You can install it by running ansible/install-ansible.sh script"
+    exit -1
```
Comment (andschwa) [inline scripts/install-open-enclave-stack]: I think this was just a typo for `exit 1`.

**Comment by oprinmarius** [comment]
@johnkord I removed the prompt from the scripts

**Comment by oprinmarius** [comment]
bors try

**Pull request merged**